### PR TITLE
SURF-504 Edited content for dropdowns

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: _component_design/dropdowns.md [hunk 1]
```diff
@@ -10,164 +10,47 @@ preview-image: components/preview-selectors.svg
...
-      <option value="Mikey">Mikey Hougland</option>
-      <option value="Tommy">Tommy Shook</option>
-      <option value="Ty">Ty Taylor</option>
-    </select>
-  </div>
-</div>
+### Basic information
 
-## Type Ahead Select
+.. REQUIRES IMAGE
```
Comment (asettle) [inline _component_design/dropdowns.md]: @araiford - as per the PDF, this section here had an image that was not included on the PDF. Could you please add to this PR?

File: _component_design/dropdowns.md [hunk 2]
```diff
@@ -10,164 +10,47 @@ preview-image: components/preview-selectors.svg
...
-      <option value="VT">Vermont</option>
-      <option value="VA">Virginia</option>
-      <option value="WA">Washington</option>
-      <option value="WV">West Virginia</option>
-      <option value="WI">Wisconsin</option>
-      <option value="WY">Wyoming</option>
-    </select>
-  </div>
-</div>
+... REQUIRES IMAGE
```
Comment (asettle) [inline _component_design/dropdowns.md]: @araiford - ditto with the comment above :)

File: _component_design/specs-states.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+title: Specs and states
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: |
+  Helix controls allow users to select options in a variety of ways including
```
Comment (asettle) [inline _component_design/specs-states.md]: @araiford - Is this usage statement correct? I made a new file, seeing as this did not previously exist (unless it did under a different name?)

File: _component_design/dropdowns.md [hunk 4]
```diff
@@ -10,164 +10,102 @@ preview-image: components/preview-selectors.svg
...
+* Keeping the drop-down pattern to a set of items that are easily parsable by a
+  user. While there is not a hard rule here for an upperbounds limit of items
+  to include a drop-down, once a drop-down data set exceeds 10 items it should
+  be looked at to determine if it could be streamlined.
+* Keeping the drop-down label viewable through all states of the dropdown. This
+  ensures users remember what label prompted the data set they are viewing.
+
+## Basic information
+
+.. REQUIRES IMAGE
```
Comment (asettle) [inline _component_design/dropdowns.md]: @araiford - missing images here and on line 56

**Comment by asettle** [comment]
@robb-romans - I was basing most of my changes off what you did too, could you double check for consistency? :)

File: _component_design/dropdowns.md [hunk 5]
```diff
@@ -10,164 +10,87 @@ preview-image: components/preview-selectors.svg
...
+
+Once a drop-down menu is expanded, the options contained within have four
+states:
+
+* Default
+* Hovered
+* Selected
+* Selected and hovered
+
+{% figure [caption:"When you need to describe an image in greater detail, use this
```
Comment (araiford) [inline _component_design/dropdowns.md]: Sorry about this -- this caption text shouldn't have been in the final PDF.  It's just placeholder text in sketch, and can be removed.

File: _component_design/dropdowns.md [hunk 6]
```diff
@@ -10,164 +10,87 @@ preview-image: components/preview-selectors.svg
...
+
+Drop-downs can be in the following states:
+
+* Default
+* Disabled
+* Focus
+* Error
+
+{% figure [caption:"When you need to describe an image in greater detail, use this block
+ of text so that you can keep the left area clutter-free. 3 lines or less."] [class:"image bg-light border"] %}
```
Comment (araiford) [inline _component_design/dropdowns.md]: Can kill this caption -- it's just placeholder text that should have been removed before the PDF was exported.

File: _component_design/dropdowns.md [hunk 7]
```diff
@@ -10,164 +10,87 @@ preview-image: components/preview-selectors.svg
...
+ ![]({{site.url}}/assets/images/components/content-areas/dropdowns/dropdowns-states.svg){:width="80%"}
+ {% endfigure %}
+
+## Option label
+
+Clicking a drop-down menu displays the options contained within the dropdown.
+
+{% figure [caption:"When you need to describe an image in greater detail, use this
+ block of text so that you can keep the left area clutter-free. 3 lines or less."] [class:"image bg-light border"] %}
```
Comment (araiford) [inline _component_design/dropdowns.md]: Same here

**Comment by araiford** [comment]
@asettle -- thanks for cranking this one out!  After removing those captions it will be 👍

**Comment by araiford** [comment]
🚢

**Pull request merged**